root.tmpl

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
Lets make a separate folder for templates.

Reply:
done in a future branch